jwst/regtest/test_nirspec_ifu.py

```diff
@@ -1,14 +1,36 @@
-"""Regression tests for NIRSpec"""
+"""Regression tests for NIRSpec IFU"""
+from pathlib import Path
 import pytest
 
+from jwst.associations import load_asn
+from jwst.lib.suffix import replace_suffix
+
 from . import regtestdata as rt
 
+# Define artifactory source and truth
+INPUT_PATH = 'nirspec/ifu'
+TRUTH_PATH = 'truth/test_nirspec_ifu'
+
 
 @pytest.fixture(scope='module')
 def run_spec2(jail, rtdata_module):
-    """Run the pipelines"""
+    """Run the Spec2Pipeline on a single exposure"""
+    rtdata = rtdata_module
+
+    # Setup the inputs
+    asn_name = 'single_nrs1_spec3_asn.json'
+    rtdata.get_data(INPUT_PATH + '/' + asn_name)
```

Comment:
50/50 here: The original intent was to have the "chained" pipelines work off of one file, which this approach uses the association as the primary input. As such, one doesn't what to use `get_asn` because the test doesn't want the `cal` file; the `run_spec2` will produce that.

The other side is to use the single `rate` file as the primary input, and build the association for the `run_spec3` fixture.

Reply:
Saying that "the test doesn't want the ``cal`` file" suggests that the ASN file in use is a level-3 ASN, where the members are in fact ``cal`` files? But this particular test module is running the ``spec2`` pipeline, so why are you using a level-3 ASN as input? Why not use an actual level-2 ("spec2") ASN that has ``rate`` file(s) as members?

Reply:
Agreed. Makes more sense to use the single file as the primary input. Will fix.